control-plane/grpc/src/operations/node/traits.rs

```diff
@@ -170,6 +177,7 @@ impl From<Node> for node::Node {
                     node_id: types_v0_state.id.to_string(),
                     endpoint: types_v0_state.grpc_endpoint.to_string(),
                     status: grpc_node_status as i32,
+                    node_nqn: types_v0_state.node_nqn.clone().map(|nqn| nqn.to_string()),
```

Comment:
```suggestion
                    node_nqn: types_v0_state.node_nqn.map(|nqn| nqn.to_string()),
```

Reply:
Right, it needs .as_ref() instead of .clone()

Reply:
Yep.